Source/Core/Core/CoreTiming.cpp

```diff
@@ -396,8 +396,7 @@ std::string GetScheduledEventsSummary()
   std::sort(clone.begin(), clone.end());
   for (const Event& ev : clone)
   {
-    text += StringFromFormat("%s : %" PRIi64 " %016" PRIx64 "\n", ev.type->name->c_str(), ev.time,
```

Comment:
Same down here for `cinttypes`

Reply:
Yes, it's still necessary for log calls within LogPendingEvents